src/rules/__tests__/require-tothrow-message.test.js

```diff
@@ -5,28 +5,80 @@ const rule = require('../require-tothrow-message');
 
 const ruleTester = new RuleTester({
   parserOptions: {
-    ecmaVersion: 6,
+    ecmaVersion: 8,
   },
 });
 
 ruleTester.run('require-tothrow-message', rule, {
   valid: [
     // String
-    "expect(() => { throw new Error('a'); }).toThrow('a');",
```

Comment:
can you keep the old tests and just add new ones covering the fixed behavior? _If_ they start failing, it's easier to fix if they are as minimal as possible

Reply:
Is there a better way to use `await` without having to wrap the whole test inside an `async function`?

Reply:
Nope (afaik)

Reply:
Do you prefer force push to keep commits clean or commit normally and you'll merge them later?

Reply:
Commit normally, please :) easier to follow, and we use squash merges

Reply:
I think I updated as you requested. Let me know if there is anything else!